libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/mongo/MongoCollection.kt

```diff
@@ -0,0 +1,61 @@
+package gov.cdc.ocio.database.mongo
+
+import gov.cdc.ocio.database.persistence.Collection
+import com.google.gson.Gson
+import org.bson.Document
+
+
+/**
+ * MongoDB Collection implementation
+ *
+ * @property mongoCollection MongoCollection<Document>
+ * @constructor
+ */
+class MongoCollection(private val mongoCollection: com.mongodb.client.MongoCollection<Document>): Collection {
+
+    /**
+     * Execute the provided query and return the results as POJOs.
+     *
+     * @param query String?
+     * @param classType Class<T>?
+     * @return List<T>
+     */
+    override fun <T> queryItems(query: String?, classType: Class<T>?): List<T> {
+        TODO("Not yet implemented")
```

Comment:
Yes, deliberately off for now.  I added it to the future enhancements.  Reason being, you need to run Atlas in order to run SQL-like queries with MongoDB.  Thought about removing MongoDB altogether, but left it in case.